projecthub/lib/projecthub/workers/project_init.ex

```diff
@@ -61,6 +61,7 @@ defmodule Projecthub.Workers.ProjectInit do
       Watchman.benchmark("#{@metric_name}.tick.duration", fn ->
         Projecthub.Models.Project
         |> where([p], p.state == ^StateMachine.initializing() or p.state == ^StateMachine.initializing_skip())
+        |> where([p], is_nil(p.deleted_at))
```

Comment:
Why do we need this? Will it be possible that project that are yet to be created already have deleted_at?